manifests/3/360/PalmInput/2.7.0.1702/360.PalmInput.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=AUSU.CRLF.7-4-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: 360.PalmInput
+PackageVersion: 2.7.0.1702
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /s # /S
+  SilentWithProgress: /s # /S
+  InstallLocation: /dir "<INSTALLPATH>"
+UpgradeBehavior: install
+FileExtensions:
+- piskn
+ReleaseDate: 2018-10-18
+Installers:
+- Architecture: x86
+  InstallerUrl: http://down1.xinshuru.com/installer/win/PalmInput_Setup.exe
```

Comment:
```suggestion
  InstallerUrl: https://dl.360safe.com/installer/web/PalmInputSetup.exe
```

Reply:
This is the old `InstallerUrl` (normal as it is a moving PR). The new one has already been used in another PR (https://github.com/microsoft/winget-pkgs/pull/210544)